### PR TITLE
Capitalized abbr. URL instead of url

### LINKED ISSUE #9116: "url" Needs to be Capitalized in Challenge
Challenge [Render Images from Data Sources](https://www.freecodecamp.com/challenges/render-images-from-data-sources) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

## How to Fix

[This line](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/a43b29489054f47d68fd79afeeae38456a14cde6/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json#L289) needs to be edited so that "url" is "URL".

## Screenshot

image

## Before Contributing

Please first read the [contributing guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before taking care of this issue. And feel free to visit the [Contributors Help](https://gitter.im/FreeCodeCamp/HelpContributors) chat room if you have any questions about helping. We're there to help.

### PR DESCRIPTION
**Pull request by kirushna-kumar**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [ ] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #9116 
#### Description

Minor typo change for abbr. URL instead of URL

### CODE REVIEW

**Comment by atjonathan** [comment]
#### :sparkles: :sparkles: **LGTM :+1:** :sparkles: :sparkles: 

:tada: Thanks for contributing @flipsyde606 and congratulations on your first PR submission to FreeCodeCamp! :tada:
